.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2.1
+
+aliases:
+  - &defaults
+    docker:
+      - image: circleci/node:12-browsers
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_PASSWORD
+    working_directory: ~/repo
+  - &save_git_cache
+    save_cache:
+      paths:
+        - .git
+      key: v1-git-{{ .Revision }}
+  - &restore_git_cache
+    restore_cache:
+      keys:
+        - v1-git-{{ .Revision }}
+        - v1-git-
+  - &save_npm_cache
+    save_cache:
+      paths:
+        - node_modules
+      key: v1-npm-{{ checksum "package-lock.json" }}
+  - &restore_npm_cache
+    restore_cache:
+      keys:
+        - v1-npm-{{ checksum "package-lock.json" }}
+        - v1-npm-
+  - &save_build_cache
+    save_cache:
+      paths:
+        - ./build
+        - ./intl
+      key: v1-build-{{ .Revision }}
+  - &restore_build_cache
+    restore_cache:
+      keys:
+        - v1-build-{{ .Revision }}
+        - v1-build-
+  - &build
+    <<: *defaults
+    resource_class: medium+
+    steps:
+      - *restore_git_cache
+      - checkout
+      # - *restore_npm_cache
```

Comment:
Now that we're using install I think this becomes useful again.

Reply:
done